### ISSUE TITLE
typescript-vue-urql typing issue with generated code for subscriptions

### ISSUE THREAD

**Issue by michael-pattern** [dotansimha/graphql-code-generator-community#403]
### Which packages are impacted by your issue?

@graphql-codegen/typescript-vue-urql

### Describe the bug

Similar to https://github.com/dotansimha/graphql-code-generator/issues/9421: the code generated by `typescript-vue-urql` is not compatible with recent version of @urql/vue. Apparently one of the property variables is optional in the generated code, but not for @urql/vue, causing some typescript errors. 

In this case, the issue is only localized to code generated for _Subscriptions_.

### Your Example Website or App

https://github.com/michael-pattern/issue-codegen-urql-vue

### Steps to Reproduce the Bug or Issue

`yarn` : install deps
`yarn generate` : use codegen to generate client code
`yarn test` : validate the client code

### Expected behavior

The generated code should match the typing required by `@urql/vue`

### Screenshots or Videos

![Screenshot 2023-09-14 at 3 36 42 PM](https://github.com/dotansimha/graphql-code-generator-community/assets/128397770/1aa09988-88dd-4f4d-8bf8-c8ed1b851795)

### Platform

- OS: macOs
- NodeJS: 20.4.0
- `graphql` version: 16.6.0
- `@graphql-codegen/*` version(s): 3.0.0-alpha-20230911210300-bd6b199b9
```
    "@graphql-codegen/cli": "^3.3.1",
    "@graphql-codegen/typescript": "^3.0.4",
    "@graphql-codegen/typescript-generic-sdk": "^3.1.0",
    "@graphql-codegen/typescript-operations": "^3.0.4",
    "@graphql-codegen/typescript-resolvers": "^3.2.1",
    "@graphql-codegen/typescript-vue-urql": "3.0.0-alpha-20230911210300-bd6b199b9",
    "@graphql-codegen/urql-introspection": "^2.2.1",
```

### Additional context

This is an extension of issue https://github.com/dotansimha/graphql-code-generator/issues/9421

**Comment by felixhromadko** [comment]
Hi, I think this got fixed in #373. However, it seems like the ci-pipeline failed on the last commit, so there is no public build of this fix yet.

**Comment by saihaj** [comment]
released [@graphql-codegen/typescript-vue-urql@3.0.0](https://github.com/dotansimha/graphql-code-generator-community/releases/tag/release-1695677511578)

**Comment by michael-pattern** [comment]
@saihaj I have come back around to test v3.0.0, and this actually still appears to be broken. Can you confirm on your side as well?

codegen via `graphql-codegen` will still generate a default value of `{}` for `options` in graphql subscription stubs.

**Comment by saihaj** [comment]
Released [@graphql-codegen/typescript-vue-urql@3.2.0](https://github.com/dotansimha/graphql-code-generator-community/releases/tag/release-1725547706606)
